README.md

```diff
@@ -1 +1,230 @@
-# ballerina-aks-deployment
\ No newline at end of file
+[![Build Status](https://travis-ci.org/ballerina-guides/ballerina-aks-deployment.svg?branch=master)](https://travis-ci.org/ballerina-guides/ballerina-aks-deployment)
+
+# Deployment Ballerina with Azure Kubernetes Service
+
+[AKS](https://azure.microsoft.com/en-us/services/kubernetes-service/) provides an easy to use hosted Kubernetes cluster service, which drastically reduces your time to setup your own infrastructure.
+
+> In this guide you will learn about building a Ballerina service and deploying it on Azure Kubernetes Service (AKS).
+
+The following are the sections available in this guide.
+
+- [What you'll build](#what-youll-build)
+- [Prerequisites](#prerequisites)
+- [Implementation](#implementation)
+- [Deployment](#deployment)
+- [Testing](#testing)
+
+## What you’ll build 
+
+In this guide, you will build a simple Ballerina service that generates an UUID each time, and you will deploy this service on AKS. 
```

Comment:
We need to include a lot more implementation details here so I think this is not sufficient. See [2] for example. Also we need to include a diagram of what you'll be implementing.

[2] - https://ballerina.io/learn/by-guide/restful-service/